website/docs/validating-admission-policy.md

```diff
@@ -112,24 +112,29 @@ spec:
         rego: |
           ...
 ```
-With this new engine and source added to the constraint template, now Gatekeeper webhook, audit, and shift-left can validate resources with these new CEL-based rules. 
+
+With this new engine and source added to the constraint template, now Gatekeeper webhook, audit, and shift-left can validate resources with these new CEL-based rules.
 
 ## Policy updates to generate Validating Admission Policy resources
 
-For some policies, you may want admission requests to be handled by the K8s Validating Admission Controller instead of the Gatekeeper admission webhook. By default, Gatekeeper is configured to generate K8s Validating Admission Policy resources if the `gatekeeper.sh/use-vap` label is used. In the event K8s Validating Admission Controller fails open, then Gatekeeper admission webhook can act as a backup. Default value for this feature flag is `--vap-enforcement=GATEKEEPER_DEFAULT`. 
+For some policies, you may want admission requests to be handled by the K8s Validating Admission Controller instead of the Gatekeeper admission webhook.
 
-Other allowed values are:
-- `NONE`: do not generate
-- `GATEKEEPER_DEFAULT`: do not generate unless label `gatekeeper.sh/use-vap: yes` is added to policy explicitly
-- `VAP_DEFAULT`: generate unless label `gatekeeper.sh/use-vap: no` is added to policy explicitly
+Gatekeeper is configured to generate K8s Validating Admission Policy (VAP) resources for all constraint templates globally if `--default-create-vap-for-templates=true` flag is set. This flag defaults to `false` at this time to not generate VAP resources by default.
+
+If you would like to override this flag's behavior for any constraint templates, you can set generateVAP explicitly on per constraint template level.
```

Comment:
Qq, is there a way to override on the constraint level still or only constraint template @JaydipGabani

Reply:
`--default-create-vapb-for-constraints`  if the flag for enabling generation of VAPB for constraint. However, to override that flag we are introducing `VAP` as an enforcement Point and a field in constraint called `scopedEnforcementActions` to override `--default-create-vapb-for-constraints`. Here is complete opt-in/opt-out of  VAP and VAPB generation behavior - [VAP-user-flow](https://docs.google.com/document/d/1UELB0dmt10K5jBwETUnSKKQN7qeivOjvi-AvzFTOsHw/edit#heading=h.6j872cv2s85o). Let me know if you have any more questions.